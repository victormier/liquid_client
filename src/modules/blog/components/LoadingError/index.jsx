import React from 'react';
import { Grid, Alert } from 'react-bootstrap';


const LoadingError = () => (
  <Grid>
    <Alert bsStyle="danger">
      <strong>Error!</strong>
      Fetching data from GraphQl server was not successful.
    </Alert>
  </Grid>
);

export default LoadingError;
