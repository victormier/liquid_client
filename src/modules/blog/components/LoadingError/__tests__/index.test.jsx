import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import LoadingError from '../index'


describe('component', () => {
  it('renders', () => {
    const component = shallow(
      <LoadingError />
    )

    expect(shallowToJson(component)).toMatchSnapshot()
  })
})
