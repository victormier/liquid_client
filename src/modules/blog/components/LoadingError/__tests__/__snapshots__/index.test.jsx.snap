// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders 1`] = `
<Grid
  bsClass="container"
  componentClass="div"
  fluid={false}
>
  <Alert
    bsClass="alert"
    bsStyle="danger"
    closeLabel="Close alert"
  >
    <strong>
      Error!
    </strong>
     Fetching data from GraphQl server was not successful.
  </Alert>
</Grid>
`;
